website/docs/diff.md

```diff
@@ -15,6 +15,8 @@ sidebar_position: 6
    * database name cannot contain capital letters.
 5. For Tigris, FerretDB requires Tigris schema validation for `msg_create`: validator must be set as `$tigrisSchemaString`.
    The value must be a JSON string representing JSON schema in [Tigris format](https://docs.tigrisdata.com/overview/schema).
+6. For Tigris, if field represents a number, an attempt to set a string that represents a suitable number 
```

Comment:
@Fashander, I need your help to phrase it better :) Do you have a moment to help me with the wording? I can show you this case so that we can describe it better.

Reply:
```suggestion
For Tigris, if a field is represented as a number, any attempt to assign a string containing a number will pass as a number (e.g. "30" for int32 field)
```